dockers/tpu-tests/tpu_test_cases.jsonnet

```diff
@@ -10,17 +10,27 @@ local tputests = base.BaseTest {
 
   timeout: 900, # 15 minutes, in seconds.
 
-  image: std.extVar('image'),
-  imageTag: std.extVar('image-tag'),
+  image: 'pytorchlightning/pytorch_lightning',
+  imageTag: 'base-xla-py{PYTHON_VERSION}-torch{PYTORCH_VERSION}',
 
   tpuSettings+: {
-    softwareVersion: 'pytorch-VERSION',
+    softwareVersion: 'pytorch-{PYTORCH_VERSION}',
   },
   accelerator: tpus.v3_8,
 
   command: utils.scriptCommand(
     |||
+      source ~/.bashrc
+      conda activate lightning
+      mkdir -p /home/runner/work/pytorch-lightning && cd /home/runner/work/pytorch-lightning
+      git clone https://github.com/PyTorchLightning/pytorch-lightning.git
       cd pytorch-lightning
+      echo $PWD
+      git ls-remote --refs origin
+      git fetch origin "refs/pull/{PR_NUMBER}/head:pr/{PR_NUMBER}" && git checkout "pr/{PR_NUMBER}"
```

Comment:
can we get here rather commit hash, at least to verify, since in extreme case in the delay between creating this docker and trigger action there can be another commit...

Reply:
Done !